docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
Delete entire file if you want the content to go on the Monitors page.

Reply:
done